### DISCUSSION TITLE
Passing OpenTracing traces via OpenTelemetry collector

### DISCUSSION THREAD

**Discussion by sasin07** [open-telemetry/opentelemetry-collector#6015] [Q&A]
Hi All,
Currently, I'm working with a vendor product that instrument the traces using OpenTracing. Is there any way to publish that generated traces to Dynatrace using the OpenTelemetry collector?

Regards,
Sasin

**Comment by jpkrohling** [comment]
You can use the Dynatrace Expoter from the contrib distribution.

https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/dynatraceexporter

cc @dyladan @arminru

**Reply by dyladan** [reply]
The Dynatrace exporter is for metrics only actually. Traces should be sent via OTLP as described in https://www.dynatrace.com/support/help/extend-dynatrace/opentelemetry/opentelemetry-traces/opentelemetry-ingest#expand--sample-collector-configuration

**Reply by sasin07** [reply]
Hi @dyladan,
Because we are working with a vendor product we can not do any code-level changes. The product generates the traces with the OpenTracing protocol. Is there any way to publish that generated Opentracing traces to the Dynatrace using the given solution?
